did/key_fuzz_test.go

```diff
@@ -0,0 +1,43 @@
+package did
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func FuzzGenerateAndDecode(f *testing.F) {
+	keytypes := GetSupportedDIDKeyTypes()
```

Comment:
First off, this is awesome; thanks for taking a stab at it!

That said, I realize now that these tests are likely better suited as unit test that go over all keytypes.

Fuzz testing works best when there is a function that can accept arbitrary random input from the fuzz arguments. 

I realize that this isn't the best case for `GenerateDIDKey`. So my suggestion would be to:
* Move the existing tests you wrote to unit tests which go over all keytypes.
* Do similar fuzz tests as what you have, but calling `CreateDIDKey` instead. You can pass in the the `publicKey []byte` as a fuzz argument. For key type, you can do the same thing you're currently doing. 
  * Make sure to verify that the `publicKey` passed in is the same as what results after calling `Decode` :)